ORBIT/phases/design/_cables.py

```diff
@@ -310,6 +311,72 @@ def _initialize_cables(self):
             name = specs["name"]
             self.cables[name] = Cable(specs)
 
+    def _get_touchdown_distance(self):
+        """
+        Returns the cable touchdown distance measured from the centerpoint of
+        the substructure.
+
+        If depth <= 60, default is 0km (straight down assumed for fixed bottom).
+        If depth > 60, default is 0.3 * depth.
+        """
+
+        _design = f"{self.cable_type}_system_design"
+        depth = self.config["site"]["depth"]
+        touchdown = self.config[_design].get("touchdown_distance", None)
+
+        if touchdown is not None:
+            self.touchdown = touchdown
+
+        else:
+            if depth <= 60:
+                self.touchdown = 0
+
+            else:
+                self.touchdown = depth * 0.3
```

Comment:
Touchdown distance is the horizontal distance from the substructure to where the cable hits the seafloor.

This is an optional user configuration with the following defaults:
- depth <= 60: 0 (doesn't assume any horizontal distance for fixed projects)
- depth > 60: "depth * 0.3". This is a somewhat arbitrary factor to get the catenary equation to solve.